### DISCUSSION TITLE
edit banner title "Draft" of website drafts

### DISCUSSION THREAD

**Discussion by fretwurst** [quarto-dev/quarto-cli#10320] [Feature Requests]
### Description

The ability to mark websites as "Draft" is great! Is it somehow possible to adjust the text of "Draft" (or could this be an enhancement)? We would like to write "beta" or "new" for individual subpages. Alternatively, we could perhaps use announcements. But currently, they only work at the website level, right?

**Comment by mcanouil** [comment]
There is no option for that right now.
Note that it's a draft feature not a versioning feature.

As a workaround you can use a small JavaScript to change this by targeting the element.

**Reply by fretwurst** [reply]
Thank you for the response and clarification. Would you suggest that I submit a feature request for a versioning feature similar to drafts and announcement banners? :-)

**Reply by mcanouil** [reply]
Versioning is something plan already but no timeline for this.
(I think there is already an issue or a discussion for this)

For instance, Quarto now uses https://prelease.quarto.org deployed from a "prelease" branch while the default website is deployed from the default branch.

**Reply by mcanouil** [reply]
FYI, for a simple banner you can place where you want:

```
include-after-body:
  - text: |
      <script type="text/javascript">
      var banner = document.createElement("div");
      banner.innerHTML = "This is a message!";
      banner.classList.add("banner");
      banner.style.fontStyle = "italic";
      var body = document.body;
      body.insertBefore(banner, body.firstChild);
      </script>
      <style type="text/css">
      .banner {
        background-color: #f0f0f0;
        color: #000;
        padding: 1em;
        text-align: center;
        z-index: 9999;
      }
      </style>
```

**Comment by andrewheiss** [comment]
This is an older post, but there's a better solution nowadays. You can override the text for the "Draft" title using [language localization options](https://quarto.org/docs/authoring/language.html#alternate-language)—"Draft" is one of [the possible language values here](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/language/_language.yml)

So you can do this now:

````
---
title: "Blah"
draft: true
language: 
  draft: "This is a draft post. Don't read it!"
---
````

…and it'll appear that way in the banner:

<img width="846" height="34" alt="image" src="https://github.com/user-attachments/assets/2a1999da-af22-4988-b9b5-60cb7fd8a9a6" />
